README.md

```diff
@@ -1 +1,36 @@
-# daily-kahoot
\ No newline at end of file
+# Daily Prebuilt + Kahoot Integration Demo
+
+This demo integrates Kahoot directly into a Daily Prebuilt video call. 
+
+![Demo screenshot description](./screenshot.png)
```

Comment:
Hoping you can update this part @adamichelle , I did not have time to grab a screenshot here yet!